test/fixtures/source-map/output/source_map_throw_async_stack_trace.mts

```diff
@@ -4,8 +4,8 @@ import '../../../common/index.mjs';
 
 interface Foo {
   /** line
-   *
-   * blocks */
+  *
+  * blocks */
```

Comment:
Feels like the idea of linting test files like this is not a good idea - don't you want to have a variety of styles tested?